### PR TITLE
okx: Add fee fields for USDC pairs

### PR DESCRIPTION
**Pull request by shazbert**
# PR Description

Add missing fee fields for USDC pairs.

Fixes # (issue)

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How has this been tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration and
also consider improving test coverage whilst working on a certain feature or package.

- [ ] go test ./... -race
- [ ] golangci-lint run
- [ ] Test X

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: exchanges/okx/okx.go [hunk 1]
```diff
@@ -2158,30 +2158,24 @@ func (ok *Okx) GetFee(ctx context.Context, feeBuilder *exchange.FeeBuilder) (flo
...
 			return 0, errors.New("no trade fee response found")
 		}
 		if feeBuilder.IsMaker {
-			if fee, err = strconv.ParseFloat(responses[0].FeeRateMaker, 64); err != nil || fee == 0 {
-				fee, err = strconv.ParseFloat(responses[0].FeeRateMakerUSDT, 64)
-				if err != nil {
-					return fee, err
-				}
+			if fee = responses[0].FeeRateMaker.Float64(); fee == 0 {
+				fee = responses[0].FeeRateMakerUSDT.Float64()
```
Comment (thrasher-) [inline exchanges/okx/okx.go:2174]: Please also add in support for USDC based on the supplied quote pair and then this is tACK!

**Review by thrasher- — CHANGES_REQUESTED** [review]

File: exchanges/okx/okx.go [hunk 2]
```diff
@@ -2158,42 +2158,36 @@ func (ok *Okx) GetFee(ctx context.Context, feeBuilder *exchange.FeeBuilder) (flo
...
 		}
 		if feeBuilder.IsMaker {
-			if fee, err = strconv.ParseFloat(responses[0].FeeRateMaker, 64); err != nil || fee == 0 {
-				fee, err = strconv.ParseFloat(responses[0].FeeRateMakerUSDT, 64)
-				if err != nil {
-					return fee, err
-				}
+			if feeBuilder.Pair.Quote.Equal(currency.USDC) {
+				fee = responses[0].FeeRateMakerUSDC.Float64()
+			} else if fee = responses[0].FeeRateMaker.Float64(); fee == 0 {
```
Comment (gloriousCode) [inline exchanges/okx/okx.go:2174]: I missed the setting in the if statement and couldn't see how fee was being set for LTC-ETH for too long 😭

**Review by gloriousCode — APPROVED** [review]
tACK for its stated purpose. Though its hardcoded for spot and pairs like BTC-USDC-230929 won't work (but are supported by the API), but that can be an adventure for another day

**Review by thrasher- — APPROVED** [review]
Thanks for making those changes!

**Pull request merged**